tests/test_django_coverage.py

```diff
@@ -0,0 +1,125 @@
+"""
+Tests for the DjangoSearch coverage functionality.
+"""
+import pytest
+from mock import DEFAULT, patch
+
+from code_annotations.find_django import DjangoSearch
+from tests.fake_models import (
+    FakeBaseModelAbstract,
+    FakeBaseModelBoring,
+    FakeBaseModelBoringWithAnnotations,
+    FakeBaseModelNoAnnotation,
+    FakeBaseModelProxy,
+    FakeBaseModelWithAnnotation,
+    FakeBaseModelWithNoDocstring,
+    FakeChildModelMultiAnnotation,
+    FakeChildModelMultiWithBrokenAnnotations,
+    FakeChildModelSingleAnnotation,
+    FakeChildModelSingleWithAnnotation
+)
+from tests.helpers import EXIT_CODE_FAILURE, EXIT_CODE_SUCCESS, call_script_isolated
+
+ALL_FAKE_MODELS = (
+    FakeBaseModelAbstract,
+    FakeBaseModelBoring,
+    FakeBaseModelBoringWithAnnotations,
+    FakeBaseModelNoAnnotation,
+    FakeBaseModelProxy,
+    FakeBaseModelWithAnnotation,
+    FakeBaseModelWithNoDocstring,
+    FakeChildModelMultiAnnotation,
+    FakeChildModelMultiWithBrokenAnnotations,
+    FakeChildModelSingleAnnotation,
+    FakeChildModelSingleWithAnnotation
+)
```

Comment:
I could try that, though then my list would be all strings that would need to be converted to types. If we needed to import something else from that file in there it might confuse things and / or break `isort`. It just seemed less confusing and better for the future to be explicit about it.